src/strategies/mantra-dao/index.ts

```diff
@@ -0,0 +1,76 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'Spaceforce';
+export const version = '0.1.0';
+
+const abi = [
+  {
+    constant: true,
+    inputs: [
+      {
+        internalType: 'address',
+        name: 'account',
+        type: 'address'
+      }
+    ],
+    name: 'balanceOf',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: '',
+        type: 'uint256'
+      }
+    ],
+    payable: false,
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
+
+export async function strategy(
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const increaceWeighting = (number) => {
+    return number * options.omUniScale
+  }
+
+  const omStakingResponse = await multicall(
```

Comment:
Can you merge the 2 multicall in one? That will avoid sending 2 requests from user browsers and make result faster to resolve.

Reply:
Ok, Can you give any ideas how to do that, i'm not familiar with multi-call. Thanks for the help!

Reply:
You will need to merge the 2 arrays of calls together:
- This one: https://github.com/snapshot-labs/snapshot.js/pull/27/files#diff-b5ff3ff9952840d980dad2da4ac1c03ae1bee0195913c96fe1f895937990f30fR48
- And this one https://github.com/snapshot-labs/snapshot.js/pull/27/files#diff-b5ff3ff9952840d980dad2da4ac1c03ae1bee0195913c96fe1f895937990f30fR55

You can use `array1.concat(array2)` for this. Then you would do a single multicall and separate the results you get in 2 using the array length. See example here: https://github.com/snapshot-labs/snapshot.js/blob/master/src/strategies/ctoken/index.ts#L92-L93